dist/completeExercise/completeExercise.js

```diff
@@ -466,8 +466,6 @@
                         return;
                     }
 
-                    $ctrl.timeEnabled = true;
-
                     if (angular.isUndefined(exerciseResult.duration)) {
```

Comment:
Restore this line, change the execution flow as we discussed (call this func from the _invoke func w. the TimeEnabled parameter